### PR TITLE
update aad group in HMCTS way

### PR DESCRIPTION
**Pull request by NickAzureDevops**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (https://tools.hmcts.net/jira/browse/DTSPO-13867) ###

update AAD group creation section on HMCT way 

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by thomast1906 — DISMISSED** [review]
?

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 1]
```diff
@@ -17,9 +17,13 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
 
 ## Creating a new team
 
-If you are unsure which groups you need to be a part of, ask your team or create a help request in the [#platops-help](https://hmcts-reform.slack.com/messages/platops-help) channel on Slack. For example:
+If you are unsure which groups you need to be a part of, ask your teammate to create a pull request to [devops-azure-ad](https://github.com/hmcts/devops-azure-ad/blob/master/users/groups.yml) and get it approved by Platform Operations.
```
Comment (timja) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:20]:
```suggestion
If you are a new team or just need a group created, then do a pull request to the [devops-azure-ad](https://github.com/hmcts/devops-azure-ad/blob/master/users/groups.yml) repository and get it approved by Platform Operations.
```

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 2]
```diff
@@ -17,9 +17,13 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
...
 
-> Hi
-> I am part of a new team "[Team name]"
+It should look like below
```
Comment (timja) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:22]:
```suggestion
It should look like:
```

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 3]
```diff
@@ -17,9 +17,13 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
...
 
-> Can you please create a team AAD group: "DTS [Team name]"
+
+```yaml
+groups:
+- name: DTS test group // this is the name of the group you want to create
```
Comment (timja) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:27]:
```suggestion
- name: DTS Your team name
```

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 4]
```diff
@@ -17,9 +17,13 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
...
+
+```it is thokjtoh
```
Comment (timja) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:29]:
```suggestion
```

```

**Review by timja — CHANGES_REQUESTED** [review]

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 5]
```diff
@@ -17,9 +17,13 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
...
-> I am part of a new team "[Team name]"
+It should look like:
 
-> Can you please create a team AAD group: "DTS [Team name]"
+
+```yaml
+groups:
+- name: DTS Your Team name
+
+```
```
Comment (timja) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:29]:
```suggestion
```

```

**Review by timja — APPROVED** [review]

**Comment by NickAzureDevops** [comment]
please approve me @thomast1906

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 6]
```diff
@@ -17,9 +17,11 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
...
 
-> Hi
-> I am part of a new team "[Team name]"
+It should look like:
 
-> Can you please create a team AAD group: "DTS [Team name]"
+
+```yaml
+groups:
+- name: DTS Your Team name
```
Comment (thomast1906) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:27]: does this not need to be closed with ```?

File: source/cloud-native-platform/onboarding/team/azuread.html.md.erb [hunk 7]
```diff
@@ -17,9 +17,14 @@ Follow the instructions on the [person onboarding guide](/cloud-native-platform/
...
+It should look like:
 
-> Can you please create a team AAD group: "DTS [Team name]"
+
+```yaml
+
+groups:
+- name: DTS Your Team name
+
+```
```
Comment (timja) [inline source/cloud-native-platform/onboarding/team/azuread.html.md.erb:30]:
```suggestion
```

```

**Pull request merged**